test/unit/cpu/main-reduce.cxx

```diff
@@ -29,7 +29,7 @@ using namespace std;
 
 #include "Compare.hxx"
 
-#include "buildIndexSet.hxx"
+#include "buildIndexSet.cxx"
```

Comment:
Here is the other huge code smell for me; this was the hack necessary to avoid compiling bis as a library